api/backend.go

```diff
@@ -316,12 +328,16 @@ func (b *StatusBackend) registerHandlers() error {
 		return hash.Hex(), err
 	})
 
-	rpcClient.RegisterHandler(params.PersonalSignMethodName, b.personalAPI.Sign)
+	rpcClient.RegisterHandler(params.PersonalSignMethodName, unsupportedMethodHandler)
 	rpcClient.RegisterHandler(params.PersonalRecoverMethodName, b.personalAPI.Recover)
```

Comment:
Do you plan to create another PR for this one?

Reply:
No. Addressed it on this PR